contributing/ci-release.txt

```diff
@@ -0,0 +1,404 @@
+           :term:`OMERO-5.1-release-integration-web`,
+           :term:`OMERO-5.1-release-training`,
+           :term:`OMERO-5.1-release-robotframework`
+
+    :jenkinsjob:`OMERO-5.1-release-integration-broken`
+
+        This job collects the OMERO.java broken test results
+
+        #. Receives TestNG results under
+           :file:`components/tools/OmeroJava/target/reports/broken` from
+           :term:`OMERO-5.1-release-integration`
+        #. Generates TestNG report
+
+    :jenkinsjob:`OMERO-5.1-release-integration-java`
+
+        This job collects the OMERO.java integration test results
+
+        #. Receives TestNG results under
+           :file:`components/tools/OmeroJava/target/reports/integration` from
+           :term:`OMERO-5.1-release-integration`
+        #. Generates TestNG report
+
+    :jenkinsjob:`OMERO-5.1-release-integration-python`
+
+        This job collects the OMERO.py integration test results
+
+        #. Receives pytest results under
+           :file:`components/tools/OmeroPy/target/reports` from
+           :term:`OMERO-5.1-release-integration`
+        #. Generates pytest report
+
+    :jenkinsjob:`OMERO-5.1-release-integration-web`
+
+        This job collects the OMERO.web integration test results
+
+        #. Receives pytest results under
+           :file:`components/tools/OmeroWeb/target/reports` from
+           :term:`OMERO-5.1-release-integration`
+        #. Generates pytest report
+
+    :jenkinsjob:`OMERO-5.1-release-training`
+
+        This jobruns the training examples
+
+        #. Receives pytest results under
+           :file:`components/tools/OmeroWeb/target/reports` from
+           :term:`OMERO-5.1-release-integration`
+        #. Generates pytest report
+
+    :jenkinsjob:`OMERO-5.1-release-robotframework`
+
+        This job runs the robot framework tests of OMERO
+
+        #. Checks out the :envvar:`RELEASE` tag of the
+           snoopycrimecop fork of openmicroscopy.git_
+        #. Runs the robot framework tests and collect the results
+
+    :jenkinsjob:`BIOFORMATS-5.1-release`
+
+        This job triggers the release of Bio-Formats (Java & C++)
+
+        #. Triggers :term:`BIOFORMATS-5.1-release-java`
+        #. Triggers :term:`BIOFORMATS-5.1-release-cpp`
+
+    :jenkinsjob:`BIOFORMATS-5.1-release-java`
+
+        This job builds the Java downloads artifacts of Bio-Formats
+
+        #. Checks out the :envvar:`RELEASE` tag of the
+           snoopycrimecop fork of bioformats.git_
+        #. |buildBF|
+        #. |fulltestBF|
+        #. |copyreleaseartifacts|
+        #. Triggers :term:`BIOFORMATS-5.1-release-downloads`
+
+    :jenkinsjob:`BIOFORMATS-5.1-release-downloads`
+
+        This job builds the Bio-Formats Java downloads page
+
+        #. Checks out the `develop` branch of
+           https://github.com/openmicroscopy/ome-release.git
+        #. Runs `make clean bf`
+
+    :jenkinsjob:`BIOFORMATS-5.1-release-cpp`
```

Comment:
There seems to be a copy & paste error in these CPP build descriptions - release-cpp and release-cpp-downloads are the same except for the first point.